specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/deviceupdate.json

```diff
@@ -821,8 +1120,197 @@
         }
       }
     },
+    "PrivateEndpointConnection": {
+      "description": "An object that represents a private endpoint connection for a device update account.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "description": "The properties of a private endpoint connection.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "PrivateEndpointConnectionProperties": {
```

Comment:
>PrivateEndpointConnectionProperties [](http://example.com/codeflow?start=5&length=35)

Please refer to common definitions "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/..."

Reply:
@chiragg4u - Is v3 safe to use? It does not exist in RPaaS branches in the private repo. We are using v2 elsewhere.

Reply:
@darkoa-msft It is safe to use in 'main', which is the branch this PR is going to get merged into. Why not pull from main back and merge into those RPaaS branches, so they have the latest definitions to work with?